modules/components/src/AdvancedFacetView/index.js

```diff
@@ -124,7 +124,7 @@ export default class AdvancedFacetView extends React.Component {
                     op: 'in',
                     content: {
                       field: path,
-                      value: value,
+                      value: typeof value === 'string' ? [value] : value,
```

Comment:
not sure I get this? numbers should not be forced into arrays?

Reply:
I guess we should check for not arrays